src/it/noSigFail-NoinMap-strict/postbuild.groovy

```diff
@@ -15,6 +15,6 @@
  */
 def buildLog = new File( basedir, 'build.log' )
 
-assert buildLog.text.contains('[ERROR] Unsigned artifact not listed in keys map: junit:junit:pom:3.8')
-assert buildLog.text.contains('[ERROR] Unsigned artifact not listed in keys map: junit:junit:jar:3.8')
+assert buildLog.text.contains('[ERROR] Unsigned artifact not listed in keys map: junit:junit:3.8')
+assert buildLog.text.contains('[ERROR] Unsigned artifact not listed in keys map: junit:junit:3.8')
```

Comment:
duplicate line

Reply:
why remove packaging info?

Reply:
I thought it would be unnecessary information, but I see now that there are situations where the packaging-info is useful. Thanks for pointing this out!